modules/javafx.graphics/src/test/java/test/com/sun/javafx/css/BitSetTest.java

```diff
@@ -47,6 +45,11 @@ public class BitSetTest {
     private final PseudoClass b = PseudoClass.getPseudoClass("b");
     private final PseudoClass c = PseudoClass.getPseudoClass("c");
 
+    @Test
+    void equalsNullShouldBeFalse() {
+        assertFalse(set.equals(null));
+    }
+
```

Comment:
Since we are here, should we add a case to test `equals()` with a non-empty `HashSet` or `Set.of()`, similar to `twoEmptyBitSetsShouldBeEqual()`?